resources/views/layout/metadata.blade.php

```diff
@@ -107,6 +107,18 @@
     <script src="/vendor/js/moment-locales/{{ $momentLocale }}.js" data-turbolinks-track="reload"></script>
 @endif
 
+@if (config('captcha.sitekey') !== '' && config('captcha.secret') !== '')
+    <script src="https://www.recaptcha.net/recaptcha/api.js?render=explicit&onload=renderCaptcha" async defer></script>
```

Comment:
should set the locale as well. And maybe use NoCaptcha's `renderJs`? Also where did you get the url from? The documentation says [`google.com`](https://developers.google.com/recaptcha/docs/display)

Reply:
This issue suggests that google.com is blocked in China: https://github.com/anhskohbo/no-captcha/pull/144

Also, https://developers.google.com/recaptcha/docs/faq
```
Can I use reCAPTCHA globally?

Yes, please use "www.recaptcha.net" in your code in circumstances when "www.google.com" is not accessible.

First, replace <script src="https://www.google.com/recaptcha/api.js" async defer></script> with <script src="https://www.recaptcha.net/recaptcha/api.js" async defer></script>
After that, apply the same to everywhere else that uses "www.google.com/recaptcha/" on your site.
```

I explicitly didn't use renderJs, because it uses google.com.

Reply:
maybe document that somewhere